internal/pkg/term/selector/selector.go

```diff
@@ -810,17 +786,21 @@ func (s *LocalWorkloadSelector) Job(msg, help string) (string, error) {
 
 }
 
-func (s *LocalWorkloadSelector) getWorkloadSelectOptions(storeWls []*config.Workload, wsWlNames []string, workloadType string) ([]prompt.Option, error) {
-	var pluralNounString string
-	switch workloadType {
-	case anyWorkloadType:
-		pluralNounString = "jobs or services"
-	case svcWorkloadType:
-		fallthrough
-	case jobWorkloadType:
-		pluralNounString = english.PluralWord(2, workloadType, "")
-	default:
-		return nil, fmt.Errorf("unrecognized workload type %q", workloadType)
+func (s *LocalWorkloadSelector) getWorkloadSelectOptions(workloadType string) ([]prompt.Option, error) {
+	pluralNounString := english.PluralWord(2, workloadType, "")
```

Comment:
> PluralWord(quantity int, singular, plural string)

I am not sure why we need this 🤔

Reply:
This reduces code duplication across what was previously three separate functions, and it's not grammatically correct to say "retrieve job from workspace" when we're talking about multiple jobs. 
Using the plural lets us pass in our existing workloadType consts (`job, service, workload`) and get back a grammatically correct error message:

`retrieve workloads from workspace: err`
or 
`retrieve jobs from store: err`